addon/styles/_frost-expand.scss

```diff
@@ -0,0 +1,56 @@
+$frost-color-expand-label-text: $frost-color-blue-1 !global;
+$frost-color-expand-scroll-border: $frost-color-lgrey-3 !global;
+$frost-color-expand-content-bg: $frost-color-lgrey-4 !global;
+$frost-color-expand-content-text: $frost-color-grey-3 !global;
+
+.frost-expand {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
Not picturing this, can you give a brief description of why space-between is the right choice?